openssl/src/ssl/mod.rs

```diff
@@ -1911,6 +1911,80 @@ impl SslContextRef {
     pub fn num_tickets(&self) -> usize {
         unsafe { ffi::SSL_CTX_get_num_tickets(self.as_ptr()) }
     }
+
+    /// Gets the current number of sessions in the internal session cache.
+    #[corresponds(SSL_CTX_sess_number)]
+    pub fn get_session_count(&self) -> i64 {
+        unsafe { ffi::SSL_CTX_sess_number(self.as_ptr()) as i64 }
+    }
+
+    /// Gets the number of started SSL/TLS handshakes in client mode
+    #[corresponds(SSL_CTX_sess_connect)]
+    pub fn get_connect_count(&self) -> i64 {
+        unsafe { ffi::SSL_CTX_sess_connect(self.as_ptr()) as i64 }
+    }
+
+    /// Gets the number of successfully established SSL/TLS handshakes in client mode
+    #[corresponds(SSL_CTX_sess_connect_good)]
+    pub fn get_connect_good_count(&self) -> i64 {
```

Comment:
Rust APIs typically exclude the `get_` prefix.

Reply:
I also usually like to mirror the OpenSSL function names in the method names, though in this case `ctx.sess_connect_good()` is a bit of a weird name, so I don't know.

Reply:
I'll remove the `get_`

I wasn't sure about the names either. I'm fine matching OpenSSL names more-exactly, but I also find them weird when they're for statistics purposes.

Reply:
`get_` removed in 164528c4948e507f18ef7aaff3763a2baa48fbe2

Let me know how you feel about the names